src/main.rs

```diff
@@ -258,7 +263,21 @@ fn create_app<'a, 'b>(
                 .takes_value(true)
                 .possible_values(&["true", "false"])
                 .default_value("true"),
-        )
+        );
+
+    #[cfg(target_arch = "x86_64")]
+    {
+        app = app.arg(
+            Arg::with_name("sgx-epc")
+                .long("sgx-epc")
+                .help(config::SgxEpcConfig::SYNTAX)
+                .takes_value(true)
+                .min_values(1)
```

Comment:
This should be `.number_of_values(1)`

Reply:
No we can provide multiple entries, something like `--sgx-epc size=32M,prefault=on size=64M,prefault=off`, with each entry being an EPC section. The overall EPC region is calculated from the VMM.

Reply:
Cool, I missed!